documentation/.markdownlint.json

```diff
@@ -1,5 +1,6 @@
 {
   "MD013": false,
+  "MD033": false,
```

Comment:
`MD033` is no inline HTML by the way.

Reply:
Might be worth writing a comment to that effect